recipes/dcap/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+sh bootstrap.sh
+
+declare -a PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    PLATFORM_FLAGS+=(--with-sysroot="${PREFIX}")
```

Comment:
PREFIX is not the sysroot

Reply:
It's not even needed so I've removed it.